tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -988,6 +1014,82 @@ public function getValue()
                 }
                 EOD,
         ];
+
+        yield 'import even if partly importable using namespace' => [
+            <<<'EOD'
+                <?php
+
+                namespace Ns;
+                use Ns\A\B;
+                use Ns\A\B\C;
+
+                new A();
+                new B();
+                new C();
+                EOD,
+            <<<'EOD'
+                <?php
+
+                namespace Ns;
+
+                new \Ns\A();
+                new \Ns\A\B();
+                new \Ns\A\B\C();
+                EOD,
+            ['import_symbols' => true],
+        ];
+
+        yield 'do not import relative symbols if not configured so - namespaced' => [
+            <<<'EOD'
+                <?php
+
+                namespace Ns;
+
+                new A();
+                new A\B();
+                new A\B\C();
+                EOD,
+            null,
+            ['import_symbols' => true],
+        ];
```

Comment:
I don't get this test case's name (and following one too). There's no config option for relative imports here (introduced in #7721?). Does it simply assume there _will be_ such option, but importing must not be done without opting-in?

Reply:
Yes, this is #7721, `A`, `A\B` are relative types, the FQCNs are `(\)Ns\A` and `(\)Ns\A\B` respestively.